docs/ui/components/Header/Search.tsx

```diff
@@ -0,0 +1,208 @@
+import { css } from '@emotion/react';
+import { theme, breakpoints, iconSize, SearchIcon } from '@expo/styleguide';
+import Router from 'next/router';
+import * as React from 'react';
+
+import * as Utilities from '~/common/utilities';
+import { paragraph } from '~/components/base/typography';
+import { LATEST_VERSION } from '~/constants/versions';
+import { SlashShortcutIcon } from '~/ui/foundations/icons';
+
+type Props = {
+  version: string;
+  mobile?: boolean;
+};
+
+// TODO(simek): ensure that processUrl is still needed
```

Comment:
Good to check indeed, I think we don't need it anymore. But maybe NextJS's router does some weird stuff with absolute URLs, as in, it probably refreshes the whole page instead of client-side redirect.

Reply:
I have removed this function after Search conversion to Functional Component, and everything seems to work correctly.